app/Services/SnipeTranslator.php

```diff
@@ -39,4 +44,28 @@ public function choice($key, $number, array $replace = [], $locale = null)
         );
     }
 
+    public function get($key, array $replace = [], $locale = null, $fallback = true)
+    {
+        $modified_locale = $locale;
+        $changed_fallback = false;
+        $previous_fallback = $this->fallback;
+        // 'legacy' translation directories tend to be two-char ('en'), not 5-char ('en-US').
+        // Here we try our best to handle that.
+        foreach (self::$legacy_translation_namespaces as $namespace) {
+            if (preg_match("/^$namespace/", $key)) {
+                $modified_locale = Helper::mapBackToLegacyLocale($locale);
+                $changed_fallback = true;
+                $this->fallback = 'en'; //TODO - should this be 'env-able'? Or do we just put our foot down and say 'en'?
```

Comment:
Do we maybe want to use the normal Laravel `FALLBACK_APP_LOCALE` env variable here? It seems like a potential footgun to invent our own override here.

Reply:
I'm okay with this as-is I think. We can always look into it again if it becomes an issue.